packages/main/src/components/DynamicPageTitle/index.tsx

```diff
@@ -125,11 +125,19 @@ const DynamicPageTitle = forwardRef((props: DynamicPageTitlePropTypes, ref: Ref<
   } = props as InternalProps;
 
   const classes = useStyles();
-  const containerClasses = clsx(classes.container, className, Device.isIE() && classes.iEClass);
   const [componentRef, dynamicPageTitleRef] = useSyncRef<HTMLDivElement>(ref);
   const [showNavigationInTopArea, setShowNavigationInTopArea] = useState(undefined);
   const isRtl = useIsRTL(dynamicPageTitleRef);
   const isMounted = useRef(false);
+  const [isPhone, setIsPhone] = useState(
+    Device.getCurrentRange(dynamicPageTitleRef.current?.getBoundingClientRect().width).name === 'Phone'
```

Comment:
```suggestion
    Device.getCurrentRange(dynamicPageTitleRef.current?.getBoundingClientRect().width)?.name === 'Phone'
```
Do we need optional chaining here?

Reply:
I thought we don't, because if the parameter of `getCurrentRange` is undefined, it should fall back to the window width, but since we use optional chaining in the other implementations, I think it's saver to add it here as well. (Maybe there where issues with the implementation if the `window` wasn't defined)